JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -65,10 +67,20 @@ module Microsoft.ApplicationInsights {
         public session: Context.Session;
 
         /**
+        * Custom properties added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public properties;
+
+        /**
+        * Custom measurements added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public measurements;
```

Comment:
Are we intentionally choosing the term `measurements`, used by the data collection schema, as opposed to the term `metrics` used by the .NET API?